File: ms_cinema/infrastructure/driven-adapters/r2dbc-postgresql/src/main/java/co/com/luisgomez29/r2dbc/genre/data/GenreData.java
package co.com.luisgomez29.r2dbc.genre.data;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.springframework.data.annotation.Id;
import org.springframework.data.relational.core.mapping.Table;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Builder(toBuilder = true)
@Table("genre")
public class GenreData {
    @Id
    private Integer id;
    private String name;
    private String description;
}

File: ms_cinema_kt/infrastructure/driven-adapters/r2dbc-postgresql/src/main/kotlin/co/com/luisgomez29/genre/data/GenreData.kt
package co.com.luisgomez29.genre.data

import org.springframework.data.annotation.Id
import org.springframework.data.relational.core.mapping.Table

@Table("genre")
data class GenreData(
    @Id
    val id: Int,
    val name: String,
    val description: String
)